ipa-core/src/protocol/ipa_prf/prf_sharding/mod.rs

```diff
@@ -864,10 +880,10 @@ pub mod tests {
         }
     }
 
-    #[derive(Debug, PartialEq)]
-    struct PreAggregationTestOutputInDecimal {
-        attributed_breakdown_key: u128,
-        capped_attributed_trigger_value: u128,
+    #[derive(Debug, Clone, Ord, PartialEq, PartialOrd, Eq)]
+    pub struct PreAggregationTestOutputInDecimal {
```

Comment:
It is still stored in a u128. I am not sure whether I would call it a decimal and whether this is substantially different from the way other numbers are stored.

Reply:
I didn't came up with that name. Rather have a separate PR for renaming things if you deem that necessary.

Reply:
this made it public which made me questioning whether it is only used in tests and should be marked as such?

Reply:
It's configured to be a test struct only with `#[cfg]`